src/ai/backend/common/events/kernel.py

```diff
@@ -229,3 +229,24 @@ def user_event(self) -> Optional[UserEvent]:
     @classmethod
     def event_name(cls) -> str:
         return "do_sync_kernel_logs"
+
+
+@dataclass
+class KernelHeartbeatEvent(BaseKernelEvent):
+    timestamp: float
```

Comment:
1. Remove timestamp here and set timestamp from event receivers
2. Add `container_id`